diff --git a/Configure.cmake b/Configure.cmake
index 7c64ce9..dcd4d96 100644
--- a/Configure.cmake
+++ b/Configure.cmake
@@ -113,7 +113,7 @@ endif()
 if(SLEEF_TARGET_PROCESSOR MATCHES "(x86|AMD64|amd64|^i.86$)")
   set(SLEEF_ARCH_X86 ON CACHE INTERNAL "True for x86 architecture.")
 
-  set(CLANG_FLAGS_ENABLE_PURECFMA_SCALAR "-mavx2;-mfma")
+  set(CLANG_FLAGS_ENABLE_PURECFMA_SCALAR "-mavx2")
 elseif(SLEEF_TARGET_PROCESSOR MATCHES "aarch64|arm64")
   set(SLEEF_ARCH_AARCH64 ON CACHE INTERNAL "True for Aarch64 architecture.")
   # Aarch64 requires support for advsimdfma4
