diff --git a/setup.py b/setup.py
index 3cab0f3..db70742 100755
--- a/setup.py
+++ b/setup.py
@@ -167,6 +167,7 @@ class build_external_clib(build_clib):
                 "/bin/sh",
                 os.path.join(os.path.realpath(local_source), "configure"),
                 "--prefix=" + build_clib,
+                "--exec-prefix=" + os.environ["WASM_LIBRARY_DIR"],
                 "--disable-curl",
                 "--with-pic",
                 "--disable-maintainer-mode",
