From a82e4b5423d72d6365a31d7cd088685b1cfc406d Mon Sep 17 00:00:00 2001
From: mrfantastic <mrfantastic@local>
Date: Sun, 26 Oct 2025 21:26:02 -0400
Subject: [PATCH] Fix

---
diff --git a/setup.py b/setup.py
index ea87da7..779f334 100644
--- a/setup.py
+++ b/setup.py
@@ -154,11 +154,6 @@ class build_clib(_build_clib):
         # Build the library
         subprocess.check_call([make_command] + make_args, cwd=build_temp)
 
-        # Check the build library
-        subprocess.check_call(
-            [make_command, "check"] + make_args, cwd=build_temp
-        )
-
         # Install the built library
         subprocess.check_call(
             [make_command, "install"] + make_args, cwd=build_temp
